main/src/ca/uwaterloo/flix/language/ast/DesugaredAst.scala

```diff
@@ -169,7 +169,9 @@ object DesugaredAst {
 
     case class Throw(exp: Expr, loc: SourceLocation) extends Expr
 
-    case class TryWith(exp: Expr, eff: Name.QName, rules: List[HandlerRule], loc: SourceLocation) extends Expr
+    case class Handler(eff: Name.QName, rules: List[HandlerRule], loc: SourceLocation) extends Expr
```

Comment:
You swapped the order. In the previous AST TryWith was before Handler. Can we stick to one order and use it consistently? Thanks :)

Reply:
handler wasnt an expression before. Handler -> RunWith makes for a better Diff